mesmerize_napari/main_offline_gui.py

```diff
@@ -402,9 +402,13 @@ def view_projections(self):
     def view_downsample_mcorr(self):
         # TODO: average set of x frames, not skip
         s = self.selected_series()
-        downsample_ratio = self.ui.spinBoxDownsampleRatio.value()
-        images = s.mcorr.get_output()[::downsample_ratio, :, :]
-        self.viewer.add_image(images)
+        subsample_ratio = self.ui.spinBoxSubsampleRatio.value()
```

Comment:
this will create a merge conflict, checkout this file from `master` within this branch?

Reply:
done. Will use downsample mcorr branch to fix